.github/workflows/mypy_primer.yaml

```
@@ -69,7 +69,7 @@ jobs:
           echo "Project selector: $PRIMER_SELECTOR"
           # Allow the exit code to be 0 or 1, only fail for actual mypy_primer crashes/bugs
           uvx \
-            --from="git+https://github.com/hauntsaninja/mypy_primer@968b2b61c05f84462d6fcc78d2f5205bbb8b98c2" \
+            --from="git+https://github.com/hauntsaninja/mypy_primer@5b695a8d2b4382d36932abcb1a06e0d54df5586b" \
```

[PR](https://github.com/hauntsaninja/mypy_primer/pull/171) has been merged:

```suggestion
            --from="git+https://github.com/hauntsaninja/mypy_primer@01a7ca325f674433c58e02416a867178d1571128" \
```